golang/lib/testsuite/test.go

```diff
@@ -35,18 +34,4 @@ type Test interface {
 	 */
 	Run(network networks.Network, testCtx TestContext)
 
-	/*
```

Comment:
Ooooooo you're doing what I've been wanting to do by moving this into the test config! I like it, though I'd recommend we not do it in this PR, with the reasons being that:
1) it won't be obvious to new users that there are two TestConfiguration values that they _must_ set (the other two, networkPartitioning and filesArtifactUrls, will correctly default to  nothing). To do this properly, I want to have a TestConfigurationBuilder that will default the timeouts to a reasonable value (e.g. 60s) 
2) doing the change will require docs updates